WooCommerce/Classes/ViewRelated/Inbox/InboxNoteRowViewModel.swift

```diff
@@ -56,7 +62,10 @@ struct InboxNoteRowViewModel: Identifiable, Equatable {
                   siteID: note.siteID,
                   stores: stores,
                   isPlaceholder: false,
-                  isRead: note.isRead)
+                  isRead: note.isRead,
+                  isSurvey: note.type == "survey",
+                  isActioned: note.status == "actioned"
```

Comment:
Just curious: is there any other type/status that we need to take care of? Will it be more scalable if we turn these into enums instead?

Reply:
Right now, the only interesting status and type for us are survey and action, but we can turn them into enums considering a `custom` status for all the unknown cases.